config/webhook/certs/tls.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
This needs to be either generated or referenced (e.g. referencing the autogenerated certs on OpenShift and reference certmanager's certificate on plain Kubernetes).